reo/src/load_profile.py

```diff
@@ -506,92 +567,77 @@ class LoadProfile(BuiltInProfile):
                  fuel_avail_before_outage=0, fuel_slope=1, fuel_intercept=0, **kwargs):
 
         self.time_steps_per_hour = time_steps_per_hour
-        self.n_timesteps = self.time_steps_per_hour*8760
-        self.percent_share_list = kwargs.get("percent_share")
+        self.n_timesteps = self.time_steps_per_hour * 8760
+        self.doe_reference_name = kwargs.get('doe_reference_name')
+        self.nearest_city = None
+        self.year = kwargs.get('year')
         # "pop"ing the following two values to replace them before calling BuiltInProfile (super class)
         doe_reference_name_list = kwargs.pop("doe_reference_name", [])
-        self.annual_kwh_list = kwargs.pop("annual_kwh", [])
-        
+        self.annual_kwh = kwargs.pop("annual_kwh", None)
+
         if user_profile:
             self.load_list = user_profile
             self.unmodified_load_list = copy.copy(self.load_list)
             self.bau_load_list = copy.copy(self.load_list)
 
-        else:  # building type and (annual_kwh OR monthly_totals_kwh) defined by user
+        else:  # building type and (annual_kwh OR monthly_totals_energy) defined by user
             if len(doe_reference_name_list) == 0:
                 message = ("Invalid LoadProfile inputs. At a minimum, please supply a loads_kw or doe_reference_name.")
                 log.error("Scenario.py raising error: " + message)
                 lp_error = LoadProfileError(task='reo.src.load_profile.py', run_uuid=dfm.run_id, user_uuid=dfm.user_id, message=message)
                 lp_error.save_to_db()
                 raise lp_error
 
-            if len(doe_reference_name_list) == 1:
-                kwargs["doe_reference_name"] = doe_reference_name_list[0]
-                if len(self.annual_kwh_list) == 1:
-                    kwargs["annual_kwh"] = self.annual_kwh_list[0]
-                super(LoadProfile, self).__init__(**kwargs)
-
-                self.load_list = self.built_in_profile
-                self.load_list_original = copy.deepcopy(self.load_list)
-
-
-                self.load_list = [val for val in self.load_list_original for _ in
-                                           range(self.time_steps_per_hour)]
-                self.unmodified_load_list = copy.copy(self.load_list)
-                self.bau_load_list = copy.copy(self.load_list)
-
-            else:  # build hybrid BuiltInProfile
+            else:
                 combine_loadlist = []
-
                 for i in range(len(doe_reference_name_list)):
-                    percent_share = self.percent_share_list[i]
                     kwargs["doe_reference_name"] = doe_reference_name_list[i]
-
-                    if self.annual_kwh_list is not None:
-                        kwargs["annual_kwh"] = self.annual_kwh_list[i]
-
+                    if self.annual_kwh is not None:
+                        kwargs["annual_energy"] = self.annual_kwh
+                        percent_share = 100
```

Comment:
@t-kwasnik this `percent_share` value and the one below on line 605 are not used. I also noticed that `percent_share` is removed from the develop load_profile.py on its line 553. Can you please check that the hybrid load profile is implemented correctly in chpbeta and the merged load_profile?

Reply:
@NLaws good catch in chpbeta lines 603, 604 and 605 can be removed, they are an artifact of any older implementation. The approach on develop works, but annual_kwh is still a list there so the logic is a little different. Both give correct results, chpbeta could be cleaned up though